kiva/gl.py

```diff
@@ -249,7 +249,6 @@ def __touch__(self, key):
                     self.__order__ = self.__order__[1:] + [key]
                 else:
                     self.__order__.append(key)
-        return
```

Comment:
No action. Just an observation that this will conflict with #392
(There are probably plenty of opportunity for merge conflicts anyway.)